src/main/kotlin/com/example/pass_manager/web/ClientController.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.pass_manager.web
+
+import com.example.pass_manager.service.ClientService
+import com.example.pass_manager.web.dto.MongoClientDto
+import com.example.pass_manager.web.dto.SpentAfterDateDto
+import com.example.pass_manager.web.mapper.MongoClientMapper.partialUpdate
+import com.example.pass_manager.web.mapper.MongoClientMapper.toDto
+import com.example.pass_manager.web.mapper.MongoClientMapper.toEntity
+import jakarta.validation.Valid
+import org.bson.types.ObjectId
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.time.Instant
+
+@RestController
+@RequestMapping("/clients")
+class ClientController(private val clientService: ClientService,) {
+
+    @GetMapping("/{id}")
+    fun calculateSpentAfterDate(
+        @RequestParam afterDate: Instant,
+        @PathVariable("id") clientId: ObjectId
```

Comment:
```suggestion
        @PathVariable("id") clientId: String,
```

Reply:
Replaced all ObjectId occurrences in controller with String